source/manage/client-proxysql-metrics.rst

```diff
@@ -34,12 +34,14 @@ The output of this command may look as follows:
      Service name: ubuntu-proxysql
 
 Beside positional arguments shown above you can specify service name and
-service address with the following flags: ``--service-name``, ``--host`` (the
-hostname or IP address of the service), and ``--port`` (the port number of the
-service). If both flag and positional argument are present, flag gains higher
-priority. Here is the previous example modified to use these flags::
+service address with the following flags: ``--service-name``, and ``--host`` (the
+hostname or IP address of the service) and ``--port`` (the port number of the
+service), or ``--socket`` (the UNIX socket path). If both flag and positional argument are present, flag gains higher
+priority. Here is the previous example modified to use these flags for both host/port or socket connections::     
 
-     pmm-admin add proxysql --username=pmm --password=pmm --service-name=my-new-proxysql --host=127.0.0.1 --port=3306 
+     pmm-admin add proxysql --username=pmm --password=pmm --service-name=my-new-proxysql --host=127.0.0.1 --port=6032
+
+     pmm-admin add proxysql --username=pmm --password=pmm --service-name=my-new-proxysql --socket=/tmp/proxysql_admin.sock
```

Comment:
I assume this is just silly linewrap and there's not a space after the -- and before socket.

Reply:
Correct.